README.md

```diff
@@ -0,0 +1,164 @@
+ti-auth.js
+==========
+
+`ti-auth.js` is a small wrapper to integrate modules into the [Travel Intelligence Portal](https://travel-intelligence.com) providing:
+- **Authentication** - unauthorized users will be redirected to the TI Portal login page and will return after successful authentication.
+- **User Context** - user information, preferences and grants are made available to the application.
+- **Utility functions** - for signout and reauthorization.
+
+
+## Installation
+via Bower:
+
+    $ bower install git@github.com:travel-intelligence/ti-auth.js --save
+
+or via NPM:
+
+    $ npm install https://github.com/travel-intelligence/ti-auth.js --save-dev
+
+Alternatively you can manually download `dist/ti-auth.js` or `dist/ti-auth.min.js` and add it to your project.
```

Comment:
I’d prefer not mentioning this given that we lose all versioning information. Especially since `master` can be in an unpublished state (even with versions/tags pointing to commits on `master` the latest commit should not be considered a stable release, as discussed).

If you really want to include this section, please update it to just provide the link to GitHub’s Releases page, here: https://github.com/travel-intelligence/ti-auth.js/releases